composer.json

```diff
@@ -14,7 +14,10 @@
         }
     ],
     "require": {
-        "php": "^7.1 || ^8.0"
+        "php": "^7.1 || ^8.0",
+        "ext-dom": "*",
+        "ext-iconv": "*",
```

Comment:
Could we depend on the symfony polyfill?

Reply:
I wonder where we need them as well tbh?